src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -396,6 +411,10 @@ class DiskImageEditor extends Component {
             id={`${idPrefix}-modal-ok`}
             bsStyle='primary'
             onClick={this.handleSave}
+            disabled={
```

Comment:
Create a new function for this check `isFormValid` or similarly named so it is easier to document and maintain.